Tests/UIKitTests/UIRefreshControlExntesionsTests.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UIRefreshControlExntesionsTests.swift
+//  SwifterSwift
+//
+//  Created by ratul sharker on 7/24/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if os(iOS)
+
+import XCTest
+@testable import SwifterSwift
+
+final class UIRefreshControlExtensionTests: XCTestCase {
+
+    func testBeginRefreshProgramatically() {
+        let tableFrame = CGRect.init(x: 0, y: 0, width: 400, height: 400)
```

Comment:
Again in here you can shorten this and other similar calls by removing `.init`. It's superfluous.